WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -574,7 +575,14 @@ public static String getLoginUrl(Blog blog) {
     public static String getDefaultUserAgent() {
         if (mDefaultUserAgent == null) {
             // TODO: use WebSettings.getDefaultUserAgent() after upgrade to API level 17+
-            mDefaultUserAgent = new WebView(getContext()).getSettings().getUserAgentString();
+            try {
+                // Catch AndroidRuntimeException that could be raised by the WebView() constructor.
+                // See https://github.com/wordpress-mobile/WordPress-Android/issues/3594
+                mDefaultUserAgent = new WebView(getContext()).getSettings().getUserAgentString();
```

Comment:
Or maybe [it's 5.0 specific](https://stackoverflow.com/questions/29575313/namenotfoundexception-webview/29809338#comment48110351_29809338)...